tests/test_mibs.py

```diff
@@ -14,14 +14,19 @@ class TestGetNextPDU(TestCase):
     @classmethod
     def setUpClass(cls):
         #For single namespace scenario, load database_config.json
-        tests.mock_tables.dbconnector.load_database_config() 
+        tests.mock_tables.dbconnector.load_database_config()
 
     def test_init_sync_d_lag_tables(self):
         db_conn = Namespace.init_namespace_dbs()
 
         lag_name_if_name_map, \
         if_name_lag_name_map, \
+<<<<<<< HEAD
+        oid_lag_name_map, \
+        lag_sai_map = Namespace.get_sync_d_from_all_namespace(mibs.init_sync_d_lag_tables, db_conn)
+=======
         oid_lag_name_map, _ = Namespace.get_sync_d_from_all_namespace(mibs.init_sync_d_lag_tables, db_conn)
+>>>>>>> 4e063e4ade89943f2413a767f24564aecfa2cd1c
```

Comment:
Remove git conflict markers

Reply:
Missed that.  Taking the HEAD branch as in https://github.com/Azure/sonic-snmpagent/pull/169/files.